Gemfile

```diff
@@ -171,7 +174,7 @@ gem 'connection_pool'
 
 gem "barnes"
 
-gem 'protected_attributes' # because we upgraded from 3
+gem 'protected_attributes_continued' # because we upgraded from 3 and then 4
```

Comment:
`protected_attributes_continued` allows us to use the old style protected attributes in modern versions of Rails.

Reply:
@caseyhelbling `protected_attributes_continued` maintains support for protected attributes in modern versions of Rails. We should move to strong_params but it's a question of priorities and testing.